{
    "type": "bundle",
    "id": "bundle--a4de50cc-e969-4ab9-b0ff-2c189531adb8",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--c51112f4-0285-431b-96ba-c9d0638cb7c3",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2023-05-27T00:04:55.911525Z",
            "modified": "2023-05-27T00:04:55.911525Z",
            "name": "CVE-2023-20882",
            "description": "In Cloud foundry routing release versions from 0.262.0 and prior to 0.266.0,a bug in the gorouter process can lead to a denial of service of applications hosted on Cloud Foundry. Under the right circumstances, when client connections are closed prematurely, gorouter marks the currently selected backend as failed and removes it from the routing pool.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2023-20882"
                }
            ]
        }
    ]
}